src/content/en/feedback/file-a-bug/index.md

```diff
@@ -0,0 +1,60 @@
+project_path: /web/_project.yaml
+book_path: /web/feedback/_book.yaml
+
+{# wf_updated_on: 2016-10-20 #}
+{# wf_published_on:  #}
```

Comment:
So… Those lines are in both of the Markdown files. Anything that needs changing there?